components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -156,14 +156,14 @@ const ContributorProfileHeader = ({
           />
 
           <Tooltip content="Get dev card">
-            <Link
+            <a
```

Comment:
Although we lose the SPA like mode by going to an anchor, this is better and it's only one page navigation.